quiche/src/recovery/gcongestion/bbr/bandwidth_sampler.rs

```diff
@@ -1386,19 +1388,13 @@ mod bandwidth_sampler_tests {
         for i in 41..=60 {
             let sample = test_sender.ack_packet(i);
             assert!(sample.state_at_send.is_app_limited, "{i}");
-            if !overestimate_avoidance || i < 60 {
-                assert!(
-                    sample.bandwidth < expected_bandwidth * 0.7,
-                    "{} {:?} vs {:?}",
-                    i,
-                    sample.bandwidth,
-                    expected_bandwidth * 0.7
-                );
-            } else {
-                // Needs further investigation: when using overestimate_avoidance,
-                // sample.bandwidth increases 1 packet earlier than expected.
```

Comment:
The fix to choose_a0_point makes this test work without the hacks added in da5379760